flink-runtime/src/main/java/org/apache/flink/runtime/io/disk/FileChannelManagerImpl.java

```diff
@@ -139,7 +139,7 @@ public void close() throws Exception {
                         .collect(Collectors.toList()));
 
         ShutdownHookUtil.removeShutdownHook(
-                shutdownHook, String.format("%s-%s", getClass().getSimpleName(), prefix), LOG);
+                shutdownHook, String.format("%s-%s", getClass().getSimpleName(), tag), LOG);
```

Comment:
If we are going to change this variable name I would call it component rather than tag, as that seems more descriptive.

Reply:
Good idea. Let me use it.